projects/dotcms-field-elements/src/components/dot-form/dot-form.e2e.ts

```diff
@@ -0,0 +1,106 @@
+import { newE2EPage, E2EPage, E2EElement } from '@stencil/core/testing';
+
+describe('dot-form', () => {
+    let page: E2EPage;
+    let element: E2EElement;
+    let formStatus = {};
+
+    const fields = [
+        {
+            fieldType: 'Text',
+            defaultValue: 'defaultValue1',
+            hint: 'hint1',
+            name: 'field1',
+            required: true,
+            value: 'value1',
+            variable: 'field1'
+        },
+        {
+            fieldType: 'Text',
+            defaultValue: 'defaultValue',
+            hint: 'hint2',
+            name: 'field2',
+            required: true,
+            value: 'value2',
+            variable: 'field2'
+        }
+    ];
+
+    beforeEach(async () => {
+        page = await newE2EPage();
+        await page.setContent(
+            `<dot-form submit-label="Save" reset-label="Reset"></dot-form>`
+        );
+        element = await page.find('dot-form');
+        element.setProperty('fields', fields);
+        await page.waitForChanges();
+        const txtFields = await element.findAll('dot-textfield');
```

Comment:
Shoulnd't the `dot-xxxx` element do this event on load?

Reply:
that's done here: 
https://github.com/dotCMS/core-web/blob/7b377e18fc57ae03a7f0da8392151c95d2dbdf7f/projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx#L73

but since the markup is set here:
https://github.com/dotCMS/core-web/blob/7b377e18fc57ae03a7f0da8392151c95d2dbdf7f/projects/dotcms-field-elements/src/components/dot-form/dot-form.e2e.ts#L32
without the `fields` attribute that can not be assign, so it needs to be set manually